src/PiWeb.Import.Sdk/IPlugin.cs

```diff
@@ -27,47 +27,48 @@ public interface IPlugin
     /// Initializes the plugin. Usually called during startup of the hosting application while showing a splash screen.
     /// Startup finishes when the returned task is completed.
     /// </summary>
-    /// <param name="context">Contains information about the environment.</param>
-    Task Init(IPluginContext context)
+    /// <param name="context">Contains information about the environment this plugin is hosted in.</param>
+    Task Init(IPluginInitContext context)

```

Comment:
Do we want to stick with `Init` here?
Usually we mark asynchronous methods with the `Async` suffix.
So to match typical naming it might be `InitAsync` or `InitializeAsync`.
Any thoughts on that?

Reply:
Yeah we can rename to InitAsync here.